tests/terraform/checks/resource/aws/example_MemoryDBSnapshotEncryptionWithCMK/main.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_memorydb_snapshot" "pass" {
+  cluster_name = aws_memorydb_cluster.example.name
+  name_prefix  = aws_memorydb_cluster.example.name
+  kms_key_arn  = aws_kms_key.example.arn
+}
+
+resource "aws_memorydb_cluster" "example" {
+  acl_name                 = "open-access"
+  name                     = "my-cluster"
+  node_type                = "db.t4g.small"
+  num_shards               = 2
+  security_group_ids       = [aws_security_group.example.id]
+  snapshot_retention_limit = 7
+  subnet_group_name        = aws_memorydb_subnet_group.example.id
+  kms_key_arn              = aws_kms_key.example.arn
+}
+
+resource "aws_memorydb_snapshot" "fail" {
+  cluster_name = aws_memorydb_cluster.example.name
+  name_prefix  = aws_memorydb_cluster.example.name
+}
```

Comment:
I couldn't find any clear info regarding the encryption of a snapshot, if the DB is encrypted. Is it really using the default key, even you encrypted the DB with your own?

Reply:
uses the cluster key 
![image](https://user-images.githubusercontent.com/1456880/151785241-dc619ce2-4cc2-48d3-be80-6274c5bfdd54.png)
So I dont think we need this check?

Reply:
either remove it or how about changing it to a YAML check and check if there is a connection between a cluster and if yes then the cluster needs to have a KMS key otherwise the snapshot needs one. I'm good with both options 😄

Reply:
ah thinks its way to niche now. Killing it with fire